dev-docs/howto/vpn/on-prem-terraform/main.tf

```diff
@@ -103,7 +107,7 @@ resource "azurerm_route_table" "route_table" {
   name                          = "vpn-routes"
   location                      = azurerm_resource_group.rg.location
   resource_group_name           = azurerm_resource_group.rg.name
-  disable_bgp_route_propagation = false
+  bgp_route_propagation_enabled = false
```

Comment:
BGP route propagation is now set to disabled.
What do we need here @burgerdev @3u13r ?

Reply:
This is not necessary for the VPN, the flip should be ok.